ansible/roles/commcarehq/templates/supervisor_pillowtop.conf.j2

```diff
@@ -0,0 +1,23 @@
+{% for pillow_name, params in pillows.get(ansible_host).items() %}
```

Comment:
Still have to remove the corresponding file from fab, right?

Reply:
https://github.com/dimagi/commcare-cloud/pull/1424/commits/ab446ad1486210dd5d5ca401b8ef7907caaf4980